SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBTrigPrimTestAlgo.cc

```diff
@@ -32,26 +32,31 @@
 
 
 
-#include <TTree.h>
-#include <TMath.h>
-
-
 //----------------------------------------------------------------------
 
-const unsigned int EcalEBTrigPrimTestAlgo::nrSamples_=5;
-const unsigned int EcalEBTrigPrimTestAlgo::maxNrTowers_=2448;
-const unsigned int EcalEBTrigPrimTestAlgo::maxNrSamplesOut_=10;
+//const unsigned int EcalEBTrigPrimTestAlgo::nrSamples_=5;
```

Comment:
delete commented-out code

Reply:
Please do, or justify why you prefer not doing it